src/Schema/Column/IntegerColumn.php

```diff
@@ -46,6 +47,10 @@ public function phpTypecast(mixed $value): ?int
             return null;
         }
 
+        if (is_string($value)) {
+            $value = rtrim($value, '.');
+        }
+
```

Comment:
```suggestion
```

Is it required?

Reply:
Let's move it into `db-mssql` driver.

Reply:
If user use this value, it also leads to unexpected warning. This may be in any DBMS.

Reply:
According to the doc "Converts the input value after retrieval from the database." we should check only values returned by DBMS. It is related with performance optimization.